src/components/existing-request/existing-request-display.vue

```diff
@@ -746,7 +744,7 @@ export default class ExistingRequestDisplay extends Mixins(
       if (paymentId) {
         // cancel the upgrade invoice
         const nrId = this.nr.id
-        await this.cancelPayment({ nrId, paymentId })
+        await NamexServices.cancelPayment(nrId, paymentId)
```

Comment:
This change needs to be verified (tested).

Reply:
was verified (flow = upgrade payment, then cancel before entering cc info)